apps/src/sites/studio/pages/pd/teacher_application/new.js

```diff
@@ -2,18 +2,52 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import TeacherApplication from '@cdo/apps/code-studio/pd/teacher_application/teacher_application.jsx';
 
-ReactDOM.render(<TeacherApplication/>, document.getElementById('application-container'));
+let schoolDistrictData = {
+  ['us-or-international']: '',
+  ['school-type']: '',
+  ['school-state']: '',
+  ['school-district']: '',
+  ['school-district-other']: false,
+  ['school']: '',
+  ['school-other']: false,
+  ['school-district-name']: '',
+  ['school-name']: '',
+  ['school-zipcode']: ''
+};
 
-$('#school-district,#school-type,#selectedCourse input').change(function () {
-  const districtValue = $('#school-district input').val();
+let districtErrorMessageHandler = (message) => {
+  document.getElementById('district-error-placeholder').innerHTML = message;
+};
 
-  if (districtValue) {
+ReactDOM.render(
+  <TeacherApplication
+    schoolDistrictData={schoolDistrictData}
+    districtErrorMessageHandler={districtErrorMessageHandler}
```

Comment:
I would probably stick to the "event handler" naming convention by calling this `onError` or something similar.